.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: go
 go:
-  - 1.7
+- 1.8
+install:
+- mkdir -p $HOME/gopath/src/k8s.io
```

Comment:
I actually find it more readable in this format, as the actual content is indented the same as if it was a map rather than a list. The [yaml spec](http://yaml.org/spec/1.2/2009-07-21/spec.html#id2576668) actually explicitly states, that this is valid yaml and very much in line with it's intentions (that spaces don't matter).